src/nupic/research/spatial_pooler.py

```diff
@@ -1663,7 +1663,7 @@ def write(self, proto):
     proto.potentialRadius = self._potentialRadius
     proto.potentialPct = self._potentialPct
     proto.inhibitionRadius = self._inhibitionRadius
-    proto.globalInhibition = self._globalInhibition
+    proto.globalInhibition = self._globalInhibition == True
```

Comment:
It seems to also take on the value of `1`, which is why I added this conversion (otherwise capnproto crashes).